Sources/XcodeProj/Objects/Targets/PBXTarget.swift

```diff
@@ -76,15 +76,28 @@ public class PBXTarget: PBXContainerItem {
     }
 
     /// Swift package product references.
-    var packageProductDependencyReferences: [PBXObjectReference]
+    var packageProductDependencyReferences: [PBXObjectReference]?
```

Comment:
Just noticed when we introduced this attribute we introduced non-determinism. I'm fixing it in this PR.